MVP/defaults-build.yaml

```diff
@@ -0,0 +1,74 @@
+- defaults:
+    name: defaults-build
+    description: |
+        Managed by Jenkins Job Builder. Do not edit via web.
```

Comment:
This non-functional desc part (inherited from ci-ops YAML) can be decided later.
Removed for now, and open for further suggestions.